python/nano/src/bigdl/nano/pytorch/trainer/Trainer.py

```diff
@@ -176,9 +175,9 @@ def quantize(self, pl_model: LightningModule,
                                     Required for static quantization.
         :param val_dataloader:      A torch.utils.data.dataloader.DataLoader object for evaluation.
         :param metric:              A torchmetrics.metric.Metric object for evaluation.
-        :param backend:             'inc' or 'nncf'('nncf' is not supported yet). Default: 'inc'.
+        :param backend:             Only 'inc' is supported. Default: 'inc'.
         :param conf:        A path to conf yaml file for quantization.
-                            Default: '', use default config.
+                            Default: None, using default config.
         :param framework:   'pytorch', 'pytorch_fx', 'pytorch_ipex'. Default: 'pytorch_fx'.
                             Consistent with Intel Neural Compressor.
         :param approach:    'static' or 'dynamic'.
```

Comment:
I think there is some typo in `accuracy_criterion`'s document we talked about before

Reply:
Fixed.